diff --git a/node_modules/@types/qs/index.d.ts b/node_modules/@types/qs/index.d.ts
index 506aae8..8d9997d 100644
--- a/node_modules/@types/qs/index.d.ts
+++ b/node_modules/@types/qs/index.d.ts
@@ -71,7 +71,7 @@ declare namespace QueryString {
         & IParseDynamicOptions<AllowDots>;
 
     interface ParsedQs {
-        [key: string]: undefined | string | ParsedQs | (string | ParsedQs)[];
+        [key: string]: undefined | string | string[];
     }
 
     function stringify(obj: any, options?: IStringifyOptions<BooleanOptional>): string;
